Resources/Prototypes/Recipes/Lathes/Packs/boombox.yml

```
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2025 SX_7 <sn1.test.preria.2002@gmail.com>
+# SPDX-FileCopyrightText: 2025 deltanedas <39013340+deltanedas@users.noreply.github.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+## Static
+
+- type: latheRecipePack
+  id: BoomboxStatic
+  recipes:
+  - BoomboxRed
+  - BoomboxGreen
+  - BoomboxBlue
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)